services/web/server/src/simcore_service_webserver/catalog/_api.py

```diff
@@ -112,7 +113,7 @@ async def dev_list_latest_services(
     return page_data, page.meta
 
 
-async def dev_get_service(
+async def get_service_v2(
```

Comment:
is this v2 really necessary?

Reply:
Yes, we still have `get_service` until I retire it in following PRs